cmd/api/src/database/auth_test.go

```diff
@@ -219,7 +220,7 @@ func TestDatabase_CreateGetUser(t *testing.T) {
 
 			newUser.Roles = newUser.Roles.RemoveByName(roleToDelete)
 
-			if err := dbInst.UpdateUser(newUser); err != nil {
+			if err := dbInst.UpdateUser(context.TODO(), newUser); err != nil {
```

Comment:
Because this isn't using a real context that we care about (currently, the root context isn't even what we're after, we just need it to grab the bheCtx off of it) I thought TODO more appropriate. Since it passes tests, I think I'd rather keep it as is, until we need a real context